### PR TITLE
Update cs0819.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/csharp/misc/cs0819.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
+
+ Multiple declarators are allowed in explicit type declarations, but not with implicitly typed variables.
+
+## To correct this error
+
+There are two options:
+1. If the variables are of the same type, use explicit declarations.
+1. Declare and assign a value to each implicitly typed local variable on a separate line.
+
+## Example 1
```
Comment (mairaw) [inline docs/csharp/misc/cs0819.md]: I don't think we need to add a number here

```suggestion
## Example
```

**Review by mairaw — COMMENTED** [review]
Overall it looks good to me, but I'll let @BillWagner confirm this.

File: docs/csharp/misc/cs0819.md (hunk 1 above)
Comment (Youssef1313) [reply docs/csharp/misc/cs0819.md]: Oh yes, I was going to add another example but forgot.
I'll add WIP and maybe add the other example tomorrow.

Comment (Youssef1313) [reply docs/csharp/misc/cs0819.md]: @mairaw, I've added the second example and removed WIP

**Review by BillWagner — APPROVED** [review]
This is great @Youssef1313 

I'll :shipit: now.

**Pull request merged**